vsts/test-release-nuget.yaml

```diff
@@ -1,237 +1,242 @@
-      IOTHUB_PFX_X509_THUMBPRINT: $(IOTHUB-PFX-X509-THUMBPRINT)
-      IOTHUB_X509_PFX_CERTIFICATE: $(IOTHUB-X509-PFX-CERTIFICATE)
-      IOTHUB_EVENTHUB_CONN_STRING_CSHARP: $(IOTHUB-EVENTHUB-CONN-STRING-CSHARP)
-      IOTHUB_EVENTHUB_COMPATIBLE_NAME: $(IOTHUB-EVENTHUB-COMPATIBLE-NAME)
-      IOTHUB_EVENTHUB_CONSUMER_GROUP: $(IOTHUB-EVENTHUB-CONSUMER-GROUP)
-      DPS_IDSCOPE: $(DPS-IDSCOPE)
-      DPS_GLOBALDEVICEENDPOINT: $(DPS-GLOBALDEVICEENDPOINT)
-      DPS_INDIVIDUALX509_PFX_CERTIFICATE: $(DPS-INDIVIDUALX509-PFX-CERTIFICATE)
-      DPS_GROUPX509_PFX_CERTIFICATE: $(DPS-GROUPX509-PFX-CERTIFICATE)
-      DPS_X509_PFX_CERTIFICATE_PASSWORD: $(DPS-X509-PFX-CERTIFICATE-PASSWORD)
-      DPS_GROUPX509_CERTIFICATE_CHAIN: $(DPS-GROUPX509-CERTIFICATE-CHAIN)
-      DPS_TPM_REGISTRATIONID: $(DPS-TPM-REGISTRATIONID)
-      DPS_TPM_DEVICEID: $(DPS-TPM-DEVICEID)
-      PROVISIONING_CONNECTION_STRING: $(PROVISIONING-CONNECTION-STRING)
-      IOTHUB_DEVICE_CONN_STRING_INVALIDCERT: $(IOTHUB-DEVICE-CONN-STRING-INVALIDCERT)
-      IOTHUB_CONN_STRING_INVALIDCERT: $(IOTHUB-CONN-STRING-INVALIDCERT)
-      DPS_GLOBALDEVICEENDPOINT_INVALIDCERT: $(DPS-GLOBALDEVICEENDPOINT-INVALIDCERT)
-      PROVISIONING_CONNECTION_STRING_INVALIDCERT: $(PROVISIONING-CONNECTION-STRING-INVALIDCERT)
-      FAR_AWAY_IOTHUB_HOSTNAME: $(FAR-AWAY-IOTHUB-HOSTNAME)
-      CUSTOM_ALLOCATION_POLICY_WEBHOOK: $(CUSTOM-ALLOCATION-POLICY-WEBHOOK)
-      IOTHUB_PROXY_SERVER_ADDRESS: 127.0.0.1:3128
-      LA_AAD_TENANT: $(LA-AAD-TENANT)
-      LA_AAD_APP_ID: $(LA-AAD-APP-ID)
-      LA_AAD_APP_CERT_BASE64: $(LA-AAD-APP-CERT-BASE64)
-      LA_WORKSPACE_ID: $(LA-WORKSPACE-ID)
-
-  - task: CopyFiles@2
-    displayName: 'Copy TRX files to the artifacts folder'
-    inputs:
-      SourceFolder: '$(Build.SourcesDirectory)'
-
-      Contents: '**/*.trx'
-
-      TargetFolder: '$(Build.ArtifactStagingDirectory)'
-
-    condition: always()
-
-  - task: CopyFiles@2
-    displayName: 'Copy ETL files to the artifacts folder'
-    inputs:
-      SourceFolder: '$(Build.SourcesDirectory)'
-
-      Contents: '**/*.etl'
-
-      TargetFolder: '$(Build.ArtifactStagingDirectory)'
-
-    condition: always()
-
-  - task: PublishBuildArtifacts@1
-    displayName: 'Publish Artifact: testresults'
-    inputs:
-      ArtifactName: testresults_windows
-
-    condition: always()
-
-  - task: PublishTestResults@2
-    displayName: 'Publish Test Results **/*.trx'
-    inputs:
-      testRunner: VSTest
-
-      testResultsFiles: '**/*.trx'
-
-      testRunTitle: 'Windows Tests'
-
-      platform: Windows
-
-      configuration: 'Debug UT + Release E2E'
-
-    condition: always()
+  - repo: self
+    clean: true
+jobs:
+  ### Linux build ###
+  - job: LINUX
+    displayName: Linux
+    strategy:
+      # Change maxParallel to 1 make builds run in serial rather than in parallel
+      maxParallel: 3
+      matrix:
+        .Net Core 2.1:
+          FRAMEWORK: netcoreapp2.1
+        .Net Core 1.1:
```

Comment:
I'd rather not add this until we have the tests running on netcoreapp1.1. This will be a false positive right now, which can be misleading for our release pipeline.

Reply:
Until https://github.com/Azure/azure-iot-sdk-csharp/pull/1249 is in